ui/src/features/Editor/useCanvasCopyPaste.ts

```diff
@@ -27,31 +30,14 @@ export default ({
   handleEdgesAdd: (newEdges: Edge[]) => void;
 }) => {
   const { copy, paste } = useCopyPaste();
+  const { toast } = useToast();
+  const t = useT();
+  const [hasItemsToPaste, setHasItemsToPaste] = useState<boolean>(false);
```

Comment:
This isn't being used anymore. Its only in a callback's deps array. Please remove

Reply:
Ah sorry my bad. Thought it was for a callback, not the custom hook's return.